html/advanced.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Cacophonator Setup</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="max-width=device-max-width, initial-scale=1">
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    <style> 
+        .row div{ 
+          display: flex; 
+          justify-content: center; 
+        } 
+        .btn-block{
+          max-width: 400px;
+        }
+    </style></head>
+
+<body>
+    {{template "navbar"}}
+
+    <div class="container pt-5 pl-0">
+        <h2 class="text-center">Advanced Settings<br></h2>
+    </div>
+    <hr>
+
+
+    <div class="container">
+            <div class="row">
+                <div class="col-xs-12 w-100" >
+                    <a class="btn btn-primary btn-block mt-3" href="/disk-memory" role="button">Disk &amp; Memory</a>
```

Comment:
This is probably the least important section for most users so let's put it last